style.css

```diff
@@ -0,0 +1,859 @@
+}
+
+/* \\\\\Header contents//// */
+.header-content-container {
+  background-image: url(img/mobile-bg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  overflow-x: hidden;
+  margin: 0 0 0 -6rem;
+  text-align: center;
+}
+
+.header-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding-top: 30%;
+  align-items: center;
+  margin-left: 3rem;
+}
+
+.content-title {
+  margin-top: -8rem;
+  align-self: center;
+  margin-left: 2.9rem;
+  font-size: 36px;
+  font-weight: 500;
+  text-align: center;
+  color: #ec5242;
+}
+
+.header-content h1 {
+  padding-left: 4rem;
+  font-weight: 700;
+  text-align: center;
+  font-family: "COCOGOOSE", sans-serif;
+  background-image: url(img/red.png);
+  background-size: cover;
+  background-position: bottom;
+  color: transparent;
+  font-size: 40px;
+  background-clip: border-box;
+  -webkit-background-clip: text;
+}
+
+.none {
+  display: none;
+}
+
+.header-content-body {
+  display: flex;
+  background-color: #f7f7f7;
+  border: 0.28rem solid #fff;
+  margin-left: 3.2rem;
+  text-align: center;
+  width: 80%;
+  font-size: 14px;
+  height: auto;
+  padding: 5px;
+}
+
+.header-content-body p {
+  font-size: 13px;
+}
+
+.summit-date {
+  margin-top: 0.5rem;
+  color: #555;
+  font-weight: 600;
+  font-size: 24px;
+  margin-left: 1.1rem;
+}
+
+.summit-venue {
+  padding-top: 0.3rem;
+  color: #555;
+  width: 70%;
+  font-size: 18px;
+  font-weight: 150;
+  text-align: center;
+}
+
+/* Main Program */
+.programs-card-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #272a31;
+  background-image: url(img/crosses-bg.png);
+  height: auto;
+  justify-content: space-around;
+  gap: 1rem;
+  color: #fff;
+  margin: 0 5%;
+}
+
+.main-program .main-program-title {
+  font-size: 24px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.hr-line {
+  color: #ec5242;
+  align-self: center;
+  border-style: solid;
+  border-width: 2px;
+  width: 25%;
+  margin-top: -0.25rem;
+}
+
+.program {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.1);
+}
+
+.program .program-title {
+  color: #ec5242;
+  font-weight: 700;
+  font-size: 1.063rem;
+}
+
+.program-description {
+  display: flex;
+  text-align: start;
+  padding: 0.2rem;
+}
+
+.see-the-whole-program p {
+  font-size: 24px;
+}
+
+.see-the-whole-program {
+  display: none;
+}
+
+.join-cs {
+  display: block;
+  width: 65%;
+  margin-left: 3.5rem;
+  height: 5rem;
+  color: #fff;
+  background-color: #ec5242;
+  font-weight: 400;
+  font-size: 1.2rem;
+  transition: 0.4s;
+  border: none;
+}
+
+.join-cs:hover,
+.join-cs:active {
+  color: #ec5242;
+  background-color: #fff;
+  border: 0.5rem solid #ec5242;
+  margin: 0.5rem;
+  margin-left: 3.5rem;
+}
+
+/* \\\\Featured section/// */
+
+#featured-speakers {
```

Comment:
- You've done extremely well, and I must commend your efforts down to this point 👏. However, you'll agree with me that the imbalance of featured speakers section got more width than the other sections of the web-page.
kindly consider fixing this bug in your design